Test/Unit/Model/Api/ShippingMethodsTest.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Test\Unit\Model\Api;
+
+use Bolt\Boltpay\Model\Api\ShippingMethods as BoltShippingMethods;
+use Magento\Framework\Phrase;
+use PHPUnit\Framework\TestCase;
+use Bolt\Boltpay\Helper\Hook as HookHelper;
+use Bolt\Boltpay\Helper\Cart as CartHelper;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\Quote\TotalsCollector;
+use Magento\Directory\Model\Region as RegionModel;
+use Magento\Framework\Exception\LocalizedException;
+use Bolt\Boltpay\Api\Data\ShippingOptionsInterfaceFactory;
+use Bolt\Boltpay\Api\Data\ShippingTaxInterfaceFactory;
+use Magento\Quote\Model\Cart\ShippingMethodConverter;
+use Bolt\Boltpay\Api\Data\ShippingOptionInterfaceFactory;
+use Bolt\Boltpay\Helper\Bugsnag;
+use Bolt\Boltpay\Helper\Log as LogHelper;
+use Magento\Framework\Webapi\Rest\Response;
+use Bolt\Boltpay\Helper\Config as ConfigHelper;
+use Magento\Framework\Webapi\Rest\Request;
+use Magento\Framework\App\CacheInterface;
+use Magento\Framework\Pricing\Helper\Data as PriceHelper;
+use Bolt\Boltpay\Model\ErrorResponse as BoltErrorResponse;
+use Bolt\Boltpay\Helper\Session as SessionHelper;
+
+/**
+ * Class ShippingMethodsTest
+ *
+ * @package Bolt\Boltpay\Test\Unit\Model\Api
+ */
+class ShippingMethodsTest extends TestCase
+{
+    /**
+     * @var BoltShippingMethods
+     */
+    private $currentMock;
+
```

Comment:
nit - Is having newline between each variable PHP standard thing? I personally find it easier to read if there is no line breaks in between

Reply:
Please, looking on core code style, as an example:
vendor/magento/module-catalog/Test/Unit/Model/LayerTest.php

In a current PSR, i could not find any info about spaces between such variables.

Reply:
PSR-2 does not offer a recommendation on this. If this is the current style across the rest of the codebase, then it's probably worth keeping it to maintain consistency.

Reply:
I think so, we should keep it.